applications/dashboard/controllers/class.importcontroller.php

```diff
@@ -96,13 +96,9 @@ public function go($transientKey = '') {
      */
     private function checkAccess() {
         $this->permission('Garden.Import'); // This permission doesn't exist, so only users with Admin == '1' will succeed.
-        \Vanilla\FeatureFlagHelper::throwIfNotEnabled(
+        \Vanilla\FeatureFlagHelper::ensureFeature(
             'Import',
-            Gdn_UserException::class,
-            [
-                t('Imports are not enabled.', 'Imports are not enabled. Set the config Feature.Import.Enabled = true to enable imports.'),
-                403
-            ]
+            t('Imports are not enabled.', 'Imports are not enabled. Set the config Feature.Import.Enabled = true to enable imports.')
```

Comment:
This should use the `StaticCacheTranslationTrait` so feature checks are more efficient.

Reply:
I'm not sure how that might apply in the context of this line.

Reply:
There is a is a translation call here. It will forget the result of its translation check and lookup again if it's just using the `t()` function. Instead if you use `StaticCacheTranslationTrait` and `self::t()` instead of `t` the translation result will be cached. This way adding a feature check into a loop somewhere will not regress our performance.